byway/src/main/webapp/destinations.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="destinations.css" >
+    <link rel="stylesheet" href= "style.css" >
+    <script src="destinations.js"></script>
+    <script src="script.js"></script>
+    <script src="https://polyfill.io/v3/polyfill.min.js?features=default"></script>
+    <script
+      src="https://maps.googleapis.com/maps/api/js?key=AIzaSyAljWlTjyagvHyZTN69Mvun2mH4Eq8j9yI&libraries=places&directions&v=weekly"
+    ></script>
+    <link href="https://fonts.googleapis.com/css2?family=Karla&family=Rubik:wght@500;700&display=swap" rel="stylesheet">
+  </head>
+  <body id="destinations-page" >
+
+    <div class="header">
+      <img src="/images/logo.png" alt="logo" class="logo">
+      <div id="name">byway.</div>
+    </div>
+    
+      
+    <div id="progressbar-container">
+      <ul id="progressbar">
+          <li>Choose Destinations</li>
+          <li>choose interests</li>
+          <li>View route</li>
+        </ul>
+    </div>
+    
+    <div id="map-search-container">
+        <div id="map"></div>
+        <div id= "destination-input-form">
+        <form action="/api/destinations" method = "post" id="user-input-form" >
+            <input
+                id="start-search-box"
+                name="start-location"
+                class="controls"
+                type="text"
+                placeholder="Enter current destination"
+                required
+            />
+             
+            <input
+                id="destinations-search-box"
+                name="destinations"
+                class="controls"
+                type="text"
+                placeholder="Enter roadtrip destinations"
+                required
+            />
+            <input type="submit" value="add city" />
+        </form>
+        <button id="user-current-address-button" onclick= "getCurrentAddress()">Use current location</button>
+        </div>
+
+        <div id="cities-container">
+            <div id="start-location"></div>
+            <div id="destinations-container"> </div>
+        </div>
+
+    </div>
+
+    <script> initAutocomplete(); setProgressBar(1); getLocations(); getStartDestination(); </script>
```

Comment:
Consider making a single `initializeDestinationsPage` function rather including every setup operation in script tags.

Also consider using `document.addEventListner('DOMContentLoaded', initializeDestinationsPage)` from the JS rather than binding the script in the HTML or using `window.onload`. https://developer.mozilla.org/en-US/docs/Web/API/Document/DOMContentLoaded_event

Reply:
done